juju/client/connection.py

```diff
@@ -22,84 +22,82 @@
 
 # Please keep in alphabetical order
 client_facades = {
-    'Action': {'versions': [2, 6, 7]},
+    'Action': {'versions': [7]},
     'ActionPruner': {'versions': [1]},
-    'Agent': {'versions': [2, 3]},
+    'Admin': {'versions': [3]},
+    'Agent': {'versions': [3]},
     'AgentLifeFlag': {'versions': [1]},
     'AgentTools': {'versions': [1]},
-    'AllModelWatcher': {'versions': [2, 3, 4]},
-    'AllWatcher': {'versions': [1, 2, 3, 4]},
+    'AllModelWatcher': {'versions': [4]},
+    'AllWatcher': {'versions': [3]},
     'Annotations': {'versions': [2]},
-    'Application': {'versions': [14, 15, 16, 17, 19]},
-    'ApplicationOffers': {'versions': [1, 2, 4]},
+    'Application': {'versions': [17, 18, 19]},
+    'ApplicationOffers': {'versions': [4]},
     'ApplicationScaler': {'versions': [1]},
-    'Backups': {'versions': [1, 2, 3]},
+    'Backups': {'versions': [3]},
     'Block': {'versions': [2]},
-    'Bundle': {'versions': [5, 6]},
-    'CharmHub': {'versions': [1]},
-    'CharmRevisionUpdater': {'versions': [2]},
-    'CharmDownloader': {'versions': [1]},
-    'Charms': {'versions': [5, 6]},
-    'Cleaner': {'versions': [2]},
-    'Client': {'versions': [5, 6]},
-    'Cloud': {'versions': [1, 2, 3, 4, 5, 7]},
-    'Controller': {'versions': [9, 11]},
-    'CrossModelRelations': {'versions': [1, 2]},
-    'CrossModelSecrets': {'versions': [1]},
-    'CrossController': {'versions': [1]},
-    'CredentialManager': {'versions': [1]},
-    'CredentialValidator': {'versions': [1, 2]},
```

Comment:
This is in a big chunk because of sorting -- all the `CAAS` facades were sorted at the end of the Cs, but should be at the start.